### PR TITLE
Fix #48: Use IssueCommentEvent's action in post msg

### LINKED ISSUE #48: If a user deletes a comment, you receive a notification for a comment creation
**Description:**
When a user is deleting comments on a PR e.g., you will be notified that the user posted a new comment:

Example

Compared to:
https://github.com/mattermost/mattermost-webapp/pull/2356

The "Comment created" from @jasonblais are actually comment deletes.

**Actual Behavior:**
Comment deletion being shown as comment creation

**Expected Behavior:**
Comment deletion being shown as comment deletion

### PR DESCRIPTION
**Pull request by checkaayush**
Fixes #48

### CODE REVIEW

**Comment by checkaayush** [comment]
@jwilander this is ready for review.

**Comment by jwilander** [comment]
Hmmm this will work but I wonder if it would be better to just not send a notification for this case? Does anybody really want notifications when someone deletes a comment?

File: server/webhook.go [hunk 1]
```diff
@@ -818,12 +818,12 @@ func (p *Plugin) handleCommentAuthorNotification(event *github.IssueCommentEvent
 	message := ""
 	switch splitURL[len(splitURL)-2] {
 	case "pull":
-		message = "[%s](%s) commented on your pull request [%s#%v](%s)"
+		message = "[%s](%s) %s comment on your pull request [%s#%v](%s)"
```
Comment (hanzei) [inline server/webhook.go]: Maybe 
```suggestion
		message = "[%s](%s) %s a comment on your pull request [%s#%v](%s)"
```

Then the message would be e.g.
`someone deleted a comment on your pull request`

**Comment by hanzei** [comment]
I agree. We should just filter out `edited` and `deleted` as `action`'s.

File: server/webhook.go (hunk 1 above)
Comment (hanzei) [reply server/webhook.go]: Let's discuss first, if we want this.

**Comment by checkaayush** [comment]
> I agree. We should just filter out `edited` and `deleted` as `action`'s.

Done with the change. Ready for final review. @hanzei

**Pull request merged**